### PR TITLE
perf : kclvm runtime memory management

### LINKED ISSUE #96: [Enhancement] KCLVM Rust core dynamic library call memory management
## Enhancement

We have a core dynamic library at the KCLVM Rust level, which has an API that can execute KCL code and return JSON/YAML results. When other languages call the API, they need the ability to release the memory allocated by the KCL code. Because at this stage KCL does not have the ability to automatically manage memory.

### PR DESCRIPTION
**Pull request by NeverRaR**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 
fix #96 

#### 2. What is the scope of this PR (e.g. component or file name):

[kclvm/runtime](https://github.com/KusionStack/KCLVM/compare/main...NeverRaR:kclvm-capi-call?expand=1#diff-5845fc16cbf2791d256eff032aec17af007f4079b3b6f2794632d0582c66c320)
[kclvm/parser](https://github.com/KusionStack/KCLVM/compare/main...NeverRaR:kclvm-capi-call?expand=1#diff-558f8c8824c0a13871b887d9c2e1873d90777d5faedb21b8d84de794c3203323)
[kclvm/ast/src/token.rs](https://github.com/KusionStack/KCLVM/compare/main...NeverRaR:kclvm-capi-call?expand=1#diff-fa011d660145a927c46918519cef2598f766b4fe2958390b803ecc5da79e7209)
[kclvm/compiler/src/codegen/llvm/node.rs](https://github.com/KusionStack/KCLVM/compare/main...NeverRaR:kclvm-capi-call?expand=1#diff-004c9803a0834e12cf4c14b73e305ed8eb0424fd818958efbca5bf639c060d3b)
[kclvm/span](https://github.com/KusionStack/KCLVM/compare/main...NeverRaR:kclvm-capi-call?expand=1#diff-9e0325450d8c55efdeff5d3793d0224456777764f14597bc99d0caaf42f8a04d)
#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

support kclvm  rust runtime memory management to reduce memory leakage

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: internal/kclvm_py/scripts/update-kclvm.sh [hunk 1]
```diff
@@ -118,10 +118,12 @@ if [ -e $topdir/kclvm/target/release/libkclvm.dll ]; then
 fi
 
 # WASM
-cargo build --release --target wasm32-unknown-unknown
-cp $topdir/kclvm/target/wasm32-unknown-unknown/release/libkclvm.a $kclvm_install_dir/lib/libkclvm_wasm32.a
-cp src/_kclvm_undefined_wasm.txt $kclvm_install_dir/lib/_kclvm_undefined_wasm.txt
+# rustup target add wasm32-unknown-unknown
+# cargo build --release --target wasm32-unknown-unknown
+# cp $topdir/kclvm/target/wasm32-unknown-unknown/release/libkclvm.a $kclvm_install_dir/lib/libkclvm_wasm32.a
+# cp src/_kclvm_undefined_wasm.txt $kclvm_install_dir/lib/_kclvm_undefined_wasm.txt
```
Comment (Peefy) [inline internal/kclvm_py/scripts/update-kclvm.sh:124]: un-comment 121-124

**Review by Peefy — APPROVED** [review]
LGTM

File: kclvm/hello.k [hunk 2]
```diff
@@ -0,0 +1 @@
+a = 1
```
Comment (Peefy) [inline kclvm/hello.k:1]: remove this file.

**Review by chai2010 — APPROVED** [review]
LGTM

**Pull request merged**